.gitignore

```diff
@@ -8,4 +8,5 @@ annotated_source/
 
 
 #Build files
-[Bb]uild/
\ No newline at end of file
+[Bb]uild/
+dist/
```

Comment:
@LeaVerou - I wonder if the build should be moved back to the `build` folder (instead of `dist`) for better compatibility with existing versions? (Could be part of #152 ). I just saw a comment in https://github.com/donmccurdy/expression-eval/issues/41 that made me wonder

Reply:
`dist` is convention though.
What if we add a file to `build` that prints out something like "JSEP has moved to using `dist` instead of `build`. Please use `dist` instead"?

Reply:
Actually, we could probably just note it in the changelog?

Reply:
We should definitely note it in the changelog!